### ISSUE TITLE
Wiki Upcoming Events/Develop Page for each ROBO Operation

### ISSUE THREAD

**Issue by mcmarti2** [FodWikis/FodWikis#25]
Develop portion of Robopedia main page to pull in upcoming events.  Develop a standard page as a template for each upcoming event.  Tie them to current Increment?  Provide individual and group training on how to make page.

**Comment by olsonjaredm** [comment]
1. Consult https://iss-www.jsc.nasa.gov/nwo/mio/riit/fpwg/web/docs/InWork.pdf for upcoming major events (dockings, EVAs). Also items from EVA's upcoming events that we care about: https://mod2.jsc.nasa.gov/wiki/EVA/index.php?title=Upcoming_Events
2. Make a page for each and add Template:Mission to populate with dates/vehicle info:
   {{Mission
   |Launch date=
   |Undock date=
   |Landing date=
   |Duration=
   |Vehicle=
   |Mission type=
   |Patch=
   |Mission synopsis=
   }}

**Comment by emanspeaks** [comment]
I think something we'd ALL like to see is this implemented on the central FOD wiki so that ALL the wikis can poll from this common content pool.  That is, if we ever figure out an answer to the Scary Transclusion Problem (tm).  (I am pretty sure we had a discussion on here about scary transcluding, but I can't find the issue now.)

FYI some of this data is maintained in parallel in a program office SharePoint page.  I had an at-length conversation with @jamesmontalvo3  about trying to tie in that parsable data somehow at one point, but I decided it wasn't worth the effort of figuring out how to do it. https://oa1.jsc.nasa.gov/MAPI-Ext/MP/SitePages/Home.aspx

**Comment by emanspeaks** [comment]
Found it: https://github.com/FodWikis/FodWikis/issues/16

**Comment by jamesmontalvo3** [comment]
Do you know how frequently the OA list of vehicle traffic is updated? They have a list of EVAs in there, too, but I wouldn't put a lot of faith in those dates.

**Comment by olsonjaredm** [comment]
From what I've seen it's updated at least every few weeks.

Marisa, I noticed these funny error triangles on the expedition pages we're making:
![image](https://cloud.githubusercontent.com/assets/13458815/9744106/814ba986-562f-11e5-98d6-3836177b4ff3.png)

I figured out that it's cuz EVA uses a separate template for expeditions (not the mission template). I imported them here:
https://mod2.jsc.nasa.gov/wiki/ROBO/index.php?title=Template%3AExpedition
https://mod2.jsc.nasa.gov/wiki/ROBO/index.php?title=Form%3AExpedition

See if changing it to this template fixes those errors.

**Comment by mcmarti2** [comment]
![expedition_form](https://cloud.githubusercontent.com/assets/13801585/9765408/b4305590-56d9-11e5-8429-6bbc3f1b257d.PNG)
Now I am getting some crazy errors when I go into edit using this form. I am not sure how to edit the start and end dates because I am not finding those sections anywhere in the expedition form or template.

**Comment by mcmarti2** [comment]
Is there somewhere I could find upcoming ROBO specific events and add those to our upcoming events?

**Comment by olsonjaredm** [comment]
Yes, two good places. One is the increment robotics plan spreadsheet: Link from the current increment sharepoint pagehttps://modspops.jsc.nasa.gov/MOD/DX/DX22/INC45_46/_layouts/xlviewer.aspx?id=/MOD/DX/DX22/INC45_46/Increment%20Robotics%20Plan/Inc_45-46_plan.xlsx&Source=https%3A%2F%2Fmodspops%2Ejsc%2Enasa%2Egov%2FMOD%2FDX%2FDX22%2FINC45%5F46%2Fdefault%2Easpx&DefaultItemOpen=1&DefaultItemOpen=1
The second is the Upcoming Ops section of the MSS Status emails – not sure if you get these from JSC-DL-Robo-Status but they essentially just reflect the same info as the increment plan in list form:

Upcoming Ops:

•        GMT 266 & 267 – RMCT swap
•        GMT 270 – HTV release OBT
•        GMT 271 – HTV release

**Comment by mcmarti2** [comment]
~~Going to start working on a ROBO ops form using the MSS operations template.~~ 
Created form. Now going to start making MSS Operation pages to add to Upcoming Events list.
